### PR TITLE
User story 10

### PR DESCRIPTION
**Pull request by martinhaiss**
- I tweaked the design with some CSS
- Got rid of the show-more/less-button
- Added more data and new images to my json file

### CODE REVIEW

File: src/components/Card.js [hunk 1]
```diff
@@ -21,69 +21,68 @@ function ActivityCard({
...
-        <Name>{name}</Name>
-        <DeleteButton
-          onClick={() => {
-            onDeleteButtonClick(id);
-          }}
-        >
-          <DeleteIcon
-            style={{ width: "20", height: "20" }}
-            aria-label="delete activity"
+      <Clickarena onClick={handleDetailsClick}>
```
Comment (mbosselmann) [inline src/components/Card.js:29]: "ClickArena" instead of "Clickarena"

File: src/components/Card.js [hunk 2]
```diff
@@ -21,69 +21,68 @@ function ActivityCard({
...
       {showDetails ? (
         <div>
+          <HorizontalLine />
           <Details>
-            <Meet>Meeting Point</Meet>
-            <Address>
-              <span style={{ display: "block" }}>{street}</span>
-              {postalcode} {city}
-            </Address>
+            <Clickarena onClick={handleDetailsClick}>
```
Comment (mbosselmann) [inline src/components/Card.js:69]: Same here

File: src/components/Card.js [hunk 3]
```diff
@@ -105,8 +104,16 @@ const Wrapper = styled.div`
...
+  h3 {
+    grid-area: headline;
+    font-size: medium;
+    margin: 10px;
+    padding: 10px;
+  }
 `;
 
+const Clickarena = styled.div``;
+
```
Comment (mbosselmann) [inline src/components/Card.js:116]: and here :)

File: src/components/Card.js [hunk 4]
```diff
@@ -158,16 +165,16 @@ const Details = styled.div`
...
-  font-size: medium;
-  margin: 10px;
-  padding: 10px;
-`;
+// const MoreInfo = styled.h3`
+//   grid-area: headline;
+//   font-size: medium;
+//   margin: 10px;
+//   padding: 10px;
+// `;
```
Comment (mbosselmann) [inline src/components/Card.js:178]: Unnecessary code?

**Review by mbosselmann — COMMENTED** [review]
Great job! 💯 I have two remarks you can consider. :)

File: src/components/Card.js (hunk 4 above)
Comment (martinhaiss) [reply src/components/Card.js:178]: Deleted, thx!

File: src/components/Card.js (hunk 1 above)
Comment (martinhaiss) [reply src/components/Card.js:29]: YouAreSoRight, thx!

File: src/components/Card.js (hunk 2 above)
Comment (martinhaiss) [reply src/components/Card.js:69]: Done!

File: src/components/Card.js (hunk 3 above)
Comment (martinhaiss) [reply src/components/Card.js:116]: Done!

**Pull request merged**